android/src/main/java/com/reactnativepagerview/PagerViewViewManager.kt

```diff
@@ -32,31 +32,34 @@ class PagerViewViewManager : ViewGroupManager<ViewPager2>() {
     //https://github.com/callstack/react-native-viewpager/issues/183
     vp.isSaveEnabled = false
     eventDispatcher = reactContext.getNativeModule(UIManagerModule::class.java)!!.eventDispatcher
-    vp.registerOnPageChangeCallback(object : OnPageChangeCallback() {
-      override fun onPageScrolled(position: Int, positionOffset: Float, positionOffsetPixels: Int) {
-        super.onPageScrolled(position, positionOffset, positionOffsetPixels)
-        eventDispatcher.dispatchEvent(
-          PageScrollEvent(vp.id, position, positionOffset))
-      }
 
-      override fun onPageSelected(position: Int) {
-        super.onPageSelected(position)
-        eventDispatcher.dispatchEvent(
-          PageSelectedEvent(vp.id, position))
-      }
+    vp.post {
```

Comment:
Here we delay adding events so as not to call onPageSelected, which always has position = 0 when creating a new viewpager

Reply:
Could you also fix it on iOS ? 
![Simulator Screen Shot - iPhone 11 - 2021-06-07 at 16 56 55](https://user-images.githubusercontent.com/12766071/121040778-779fce80-c7b2-11eb-9dc8-c10cc54ca0f2.png)